extensions/helix/package.build.ts

```diff
@@ -274,6 +274,225 @@ export const pkg = (modules: Builder.ParsedModule[]) => ({
             },
           },
         },
+
+      "space": {
+        title: "Space",
+        items: {
+          "f": { text: "Open file picker", command: "workbench.action.quickOpen" },
+          "F": {
+            text: "Open file picker at current working directory",
+            command: "dance.run",
+            args: [
+              {
+                code: [
+                  `
+                  const editor = vscode.window.activeTextEditor;
+                  const fallback = () => vscode.commands.executeCommand(
+                      'workbench.action.quickOpen',
+                    );
+
+                  if (!editor) {
+                    return await fallback();
+                  }
+
+                  const currentFileUri = editor.document.uri;
+                  const currentDirectoryUri = vscode.Uri.joinPath(currentFileUri, '..');
+
+                  const workspaceFolder = vscode.workspace.getWorkspaceFolder(currentFileUri);
+                  if (!workspaceFolder) {
+                    return await fallback()
+                  }
+
+                  const relativeDirectoryPath = vscode.workspace.asRelativePath(
+                      currentDirectoryUri,
+                      false
```

Comment:
nit: please put argument names before literals
```suggestion
                      /*foo=*/ false
```